Add part of pterodox (#292)
pterodactyl/documentation: community/customization/wings.md

# Customizing Wings
:::warning Production

> We highly recommend **NOT** performing these actions on production

Reviewer: It should probably be "on the production"

:::

> In order to make changes to wings we recommend forking it to keep your own git version making upgrades easier. <br />

Reviewer: This is a confusing sentence at least in my head, it probably requires rephrasing or at least a comma. If I understood right, this is what is meant (not a suggestion to replace with that sentence):

"We recommend creating a fork of the repo before making changes to wings to make future upgrades easier"

:::tip Editing Files

> We do not provide a guide at the current time on what files to edit to get certain results. This guide expects a basic knowledge of the `go` language

Reviewer 2: Should be called "Go" everywhere

:::

## Building Wings

> :::tip Build output

Reviewer 2: ```suggestion
:::tip
```

> GO builds to be run on the build system therefore you **HAVE TO** run the build command on a linux system

Reviewer 2: ```suggestion
By default, Go targets the system it is executed on. Therefore, the easiest way of building binaries for Linux is to execute the following steps on a Linux system.
```

:::

### Build Requirements

> GoLang compiler/runtime [https://golang.org/doc/install](https://golang.org/doc/install)

Reviewer 2: ```suggestion
You need to have an up-to-date version of the Go tools installed. See the [official instructions](https://golang.org/doc/install) for help with setting those up.
```

### Building

> Run the following command in your code directory

Reviewer 2: ```suggestion
Running the following command in your local clone of the repository to compile wings into a binary.
```

```bash
go build
```
You should now have a `wings` binary file in your wings directory.
## Install the new binary

> :::tip User Access

Reviewer 2: ```suggestion
:::tip Root required
```

> Running the following command in your local clone of the repository to compile wings into a binary.

Reviewer: Run/execute/type instead of running?

> In order to use the guide bellow you need sudo permission level you can atchive this by placing `sudo` in front of each command or running `sudo su -` in order to become root

Reviewer 2: ```suggestion
In order to execute the next few commands, you will need root permissions. If you are not logged in as root, either use `sudo` in front of each command, or run `sudo -i` to switch to a root session.
```

:::

1. Backup the current installation of wings

```bash
# Stop Wings
systemctl stop wings
# Backup Wings

> mv /usr/local/bin/wings /usr/local/bin/wingsOld

Reviewer 2: ```suggestion
mv /usr/local/bin/wings /usr/local/bin/wings-backup
```

> In order to execute the next few commands, you will need root permissions. If you are not logged in as root, either use `sudo` in front of each command, or run `sudo -i` to switch to a root session.

Reviewer: "In order to" can always be replaced with "To" which serves the same point and reduces the amount of text to read

```
2. Place the new binary in `/usr/local/bin`

> 3. Ensure aproperiate permissions on the binary `chmod u+x /usr/local/bin/wings`

Reviewer: It should be "appropriate"
